cassandra/pom.xml

```diff
@@ -27,13 +27,14 @@
     </parent>
 
     <groupId>org.apache.zeppelin</groupId>
-    <artifactId>zeppelin-cassandra_2.10</artifactId>
+    <artifactId>zeppelin-cassandra_${scala.binary.version}</artifactId>
```

Comment:
I'm not sure this should matter:

- If you're downloading the lib from maven, and use Scala 2.11, then all submodules would be 2.11 (as expected)

- If you're building from source, the artifact names would retain your _base_ project scala version number, which wouldn't really affect compatibility with anything.

Other than naming the artifacts, I don't see any other place where binary.version is used/useful.

WDYT? Let me know if you think these should match the internal version for any reason, though, and I can copy+pasta some more